pkg/serve/index.html

```diff
@@ -120,7 +120,31 @@ <h1 class="title">Skenario</h1>
             </div>
 
             <hr>
+            <div class="field is-horizontal">
+                <div class="field-label is-normal">
+                    <label class="label" for="requestTimeout">Request timeout (seconds)</label>
+                </div>
+                <div class="control">
+                    <input type="number" style="width: 5em" id="requestTimeout" value="10.0" min="1" step="1"/>
+                </div>
+            </div>
 
+            <div class="field is-horizontal">
+                <div class="field-label is-normal">
+                    <label class="label" for="requestCPUUtilization">Request CPU utilization </label>
+                </div>
+                <div class="control">
+                    <input type="number" style="width: 5em" id="requestCPUUtilization" value="200.0" min="1" step="1"/>
+                </div>
+            </div>
+            <div class="field is-horizontal">
+                <div class="field-label is-normal">
+                    <label class="label" for="requestIOUtilization">Request IO utilization </label>
```

Comment:
Same for IO.  We want to specify the time (millis) we will spend waiting on downstream services / disk / whatever for each request.

IMO we should run CPU and IO in serial to get non-binary utilization values.  E.g. a request with a requirements of 500 millicores of CPU and 500 milliseconds of IO running on a 1 core pod alone will complete in 1 second.

Reply:
Changed it. Please see e91267565b8ba92593c89fe969f1d65a35e99baf and 935d1a2c2d76ab2acf813d2b062574cf7d349078 commits . The current CPU model works like you suggest.